crates/iota-sdk/src/lib.rs

```diff
@@ -166,6 +166,7 @@ pub struct IotaClientBuilder {
     ws_url: Option<String>,
     ws_ping_interval: Option<Duration>,
     basic_auth: Option<(String, String)>,
+    custom_tls_config: Option<rustls::ClientConfig>,
```

Comment:
I would personally just call it `tls_config`, all builder fields are custom by nature. I know you probably derived the name from `with_custom_cert_store` but it's slightly over the top IMO. I don't feel very strongly about it though.